agent/app/agent.go

```diff
@@ -214,6 +216,14 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 	imageManager engine.ImageManager,
 	client api.ECSClient) int {
 
+	// Conditionally create '/ecs' cgroup root
+	if agent.cfg.TaskCPUMemLimit.Enabled() {
+		if err := agent.cgroupInit(); err != nil {
+			seelog.Criticalf("Cgroup init error: %v", err)
```

Comment:
Can you make this more descriptive and user friendly? Something on the lines `"Unable to initialize cgoup root for ECS: .."`